src/pip/_internal/utils/glibc.py

```diff
@@ -6,14 +6,12 @@
 from typing import Optional, Tuple
 
 
-def glibc_version_string():
-    # type: () -> Optional[str]
+def glibc_version_string() -> Optional[str]:
```

Comment:
Unrelated to this PR, this just reminds me we should do something with `packaging`’s manylinux/musllinux logic and unify this libc version detection logic into a common implementation.